tests/conftest.py

```diff
@@ -38,13 +38,13 @@ def notebook_service(docker_ip, docker_services):
     docker_services.wait_until_responsive(
         timeout=30.0, pause=0.1, check=lambda: is_responsive(url)
     )
-    return url, token
+    return url, token, docker_compose
 
 
 @pytest.fixture(scope="function")
 def selenium_driver(selenium, notebook_service):
     def _selenium_driver(nb_path):
-        url, token = notebook_service
+        url, token, _ = notebook_service
```

Comment:
I would use the aiidalab_exec fixture from aiidalab-docker-stack

https://github.com/aiidalab/aiidalab-docker-stack/blob/e89338181ada44420051e775645fdea00f91cff3/tests/conftest.py#L46